org.eclipse.jdt.ls.core/src/com/google/googlejavaformat/java/GoogleJavaFormatter.java

```diff
@@ -0,0 +1,130 @@
+package com.google.googlejavaformat.java;
```

Comment:
Currently this file is directly copied from https://github.com/google/google-java-format/blob/master/eclipse_plugin/src/com/google/googlejavaformat/java/GoogleJavaFormatter.java, since the google-java-format plugin is not available in the maven central.